kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/status/modem/ModemGpsMode.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ ******************************************************************************/
+
+package org.eclipse.kura.net.status.modem;
+
+/*
+ * The GPS mode supported by the modem.
+ */
```

Comment:
Please add `@since 3.0` annotation.

Reply:
I'll add it as soon as we clarify things on https://github.com/eclipse-kura/kura/pull/5414#discussion_r1787630255

Reply:
Addressed in 5576993084626dded6d1652c873310e7f10f1d9c